Formula/emscripten.rb

```diff
@@ -46,12 +47,12 @@ class Emscripten < Formula
 
   def install
     ENV.cxx11
-    # OSX doesn't provide a "python2" binary so use "python" instead.
-    python2_shebangs = `grep --recursive --files-with-matches ^#!/usr/bin/.*python2$ #{buildpath}`
+    # rewrite hardcoded paths from system python to homebrew python
+    python2_shebangs = `grep --recursive --files-with-matches ^#!/usr/bin/python #{buildpath}`
     python2_shebang_files = python2_shebangs.lines.sort.uniq
     python2_shebang_files.map! { |f| Pathname(f.chomp) }
     python2_shebang_files.reject! &:symlink?
-    inreplace python2_shebang_files, %r{^(#!/usr/bin/.*python)2$}, "\\1"
+    inreplace python2_shebang_files, %r{^#!/usr/bin/python2?$}, "#!#{HOMEBREW_PREFIX}/bin/python2"
```

Comment:
`"#!#{Formula["python@2"].opt_bin}/python2"`

Reply:
done, although there a still a lot of shebangs with `#!/usr/bin/env python2?` in `emscripten`. Let's hope that that nobody will unlink `python@2` or they will end up with a mixed python setup again.